orange_silver_2.rb

```diff
@@ -0,0 +1,72 @@
+#defining our Orange Tree Class
+class OrangeTree
+  #assigning read/write abilities to all of the attributes
+  attr_reader :age, :is_dead, :max_age, :height, :count
+
+  def initialize
+    @age = 0
+    @height = 0
+    @count = 0
+    @max_age = 10
+    @is_dead = false
+  end
+
+  def one_year_passes
+    @age += 1
+    @height += 4
+    @count = 0
+    grow_oranges
+    if @age > @max_age
+      @is_dead = true
+      @count = 0
```

Comment:
It's nice to be clear that a dead tree has no oranges, but I don't think `grow_oranges` is going to change `count` from line 17.